### PR TITLE
fix: scene metrics calculations

### PR DESCRIPTION
**Pull request by pbosio**
Metrics were not being calculated correctly.

### CODE REVIEW

**Review by pravusjif — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Tests/SceneTests.cs [hunk 1]
```diff
@@ -19,65 +19,68 @@ public class SceneTests : TestsBase
...
-            var scenesToLoad = (Resources.Load("TestJSON/SceneLoadingTest") as TextAsset).text;
+            yield return InitScene();
+            DCL.Configuration.Environment.DEBUG = true;
 
+            sceneController.SetDebug();
             yield return new WaitForSeconds(0.1f);
 
-            sceneController.UnloadAllScenes();
-
             yield return new WaitForSeconds(0.1f);
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Tests/SceneTests.cs]: Why is this statement twice here? Maybe we should remove it.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Tests/SceneTests.cs [hunk 2]
```diff
@@ -19,65 +19,68 @@ public class SceneTests : TestsBase
         [UnityTest]
         public IEnumerator PerformanceLimitControllerTests()
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Tests/SceneTests.cs:20]: I think we are lacking some support + tests regarding `ParametrizedShape` edge cases. Remember they can switch meshes around without passing through the standard `MeshesInfo` flow (the `ParametrizedShape.UpdateRenderer` lines I showed you the other day)

**Review by BrianAmadori — CHANGES_REQUESTED** [review]

**Review by javiermaria — APPROVED** [review]

**Review by pravusjif — APPROVED** [review]

**Review by BrianAmadori — APPROVED** [review]
